docs/book/troubleshooting.md

```diff
@@ -0,0 +1,76 @@
+# Troubleshooting
```

Comment:
Can we also add instruction on how to retain logs in case system pods are crashing?

Reply:
That is a deprecated command on Kubelet and it is supported by Kubernetes. I don't think we can officially support that statement by writing in our docs.